src/o365/spo/commands/web/web-get.spec.ts

```diff
@@ -62,11 +39,10 @@ describe(commands.WEB_GET, () => {
 
   after(() => {
     Utils.restore([
-      appInsights.trackEvent,
-      auth.getAccessToken,
       auth.restoreAuth,
       request.get
```

Comment:
`request.get` is already being restored in `afterEach` so it's not necessary to include it here again.